src/box/sql/util.c

```diff
@@ -1047,7 +1047,7 @@ sql_div_int(int64_t lhs, bool is_lhs_neg, int64_t rhs, bool is_rhs_neg,
 		if (u_res > (uint64_t) INT64_MAX + 1)
 			return -1;
 		*is_res_neg = u_res != 0;
-		*res = -u_res;
+		*res = u_res == (uint64_t)INT64_MAX + 1 ? INT64_MIN : -u_res;
```

Comment:
We agreed (in the SVACE chat) that all sane architectures use binary compliment (AFAIR somebody mentioned that this is even required by the C++ standard). So there's no need to fix this issue.

Reply:
Relevant SVACE issue is closed as "Won't fix".